assembl/static2/css/components/posts.scss

```diff
@@ -160,6 +163,7 @@ $deleted-post-border: 1px solid $grey1 !default;
             color: $grey2;
             font-size: $font-size-xs;
             padding: 0 0 0 10px;
+            font-family: 'LatoLight'
```

Comment:
You want to use variables instead, so that if a custom theme changes the variable, these styles will be affected. Use `$font-family4` here, instead.